flask/tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from app import *
+
+class TestConfig(config.Config):
+    """
+    Pytest config settings.
+    Uses MySQL database called "st2020testing" for adding/removing test data.
+    """
+    FLASK_ENV = 'development'
+    DEBUG = True
+    SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://admin:admin@localhost/st2020testing'
```

Comment:
Use `os.getenv('ST_DATABASE_URI_TESTING', fallback)` like the main config to allow environment overrides